Improve README (#418)
ansible-collections/community.docker: README.md

## External requirements

> Some modules and plugins require the Docker CLI programs or other external programs, some require the [Docker SDK for Python](https://pypi.org/project/docker/), and some use [requests](https://pypi.org/project/requests/) to directly talk to the Docker daemon's API. All modules and plugins require Python 2.7 or newer. Python 2.6 is no longer supported; use community.docker 2.x.y if you need to use Python 2.6.

Reviewer: ```suggestion
Some modules and plugins require Docker CLI, or other external, programs. Some require the [Docker SDK for Python](https://pypi.org/project/docker/) and some use [requests](https://pypi.org/project/requests/) to directly communicate with the Docker daemon API. All modules and plugins require Python 2.7 or later. Python 2.6 is no longer supported; use community.docker 2.x.y if you need to use Python 2.6.
```

Reviewer: A few nits for structure and clarity. Also s/talk/communicate to avoid anthropomorphic language.

> Installing the Docker SDK for Python also installs the requirements for the modules and plugins that use `requests`. If you do not want to install the SDK but install directly the required Python libraries, you need the following ones:

Reviewer: ```suggestion
Installing the Docker SDK for Python also installs the requirements for the modules and plugins that use `requests`. If you want to directly install the Python libraries instead of the SDK, you need the following ones:
```
